src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -811,7 +823,7 @@ private enum TaskStatus
             var backupStatus = GetBackupStatus(configuration.TaskId);
             var whoseTaskIsIt = _database.WhoseTaskIsIt(topology, configuration, backupStatus, keepTaskOnOriginalMemberNode: true);
             if (whoseTaskIsIt == null)
-                return TaskStatus.Disabled;
+                return TaskStatus.ClusterDown;
```

Comment:
Would the backup occur on a `Passive` node?

Reply:
No, before running the backup we check if task is ActiveByCurrentNode